lib/forms/field-helpers.js

```diff
@@ -229,14 +229,12 @@ function shouldBeRevealedBySite(sites, currentSlug) {
  * @return {boolean}
  */
 function shouldBeRevealedByData(data, operator, value, values) {
-  if (value) {
+  if (value !== undefined) {
```

Comment:
I think `typeof value !== "undefined"` is the canonical way to do this

Reply:
oh right, because `undefined` can be overwritten. <3 javascript